PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,8 @@
-#### What does this PR do?
+#### Which issue is fixed by this PR? If not related to an existing issue, what does this PR do?
```

Comment:
As long as we're doing this, can we make this subheadings instead of subsubsubheadings? i.e. `#### Some stuff` -> `## Some Stuff`. The PR title should be the main heading, but headings shouldn't skip levels.

Reply:
That seems like a good change to make

Reply:
sure. can u make that change anna?